src/general.js

```diff
@@ -78,7 +78,7 @@ function guessUserCountry() {
 
```

Comment:
We should also try to fallback to the language if we can not get the country from the language string, because only a language is reported (e.g. `de` instead of `de-de` as seen in #193).